packages/greenbox-script/src/iotms.ts

```diff
@@ -0,0 +1,474 @@
+        11: recordWorkshedIOTM($item`unopened diabolic pizza cube box`, $item`diabolic pizza cube`),
+        12: recordFamiliarIOTM($familiar`Red-Nosed Snapper`),
+    },
+    2020: {
+        1: recordItemIOTM($item`unopened Bird-a-Day calendar`, $item`Bird-a-Day calendar`),
+        2: recordItemIOTM($item`mint-in-box Powerful Glove`, $item`Powerful Glove`),
+        3: recordGardenIOTM($item`Better Shrooms and Gardens catalog`, $item`packet of mushroom spores`),
+        4: recordFamiliarIOTM($familiar`Left-Hand Man`),
+        5: recordItemIOTM($item`Guzzlr application`, $item`Guzzlr tablet`),
+        6: recordItemIOTM($item`bag of Iunion stones`, $item`Iunion Crown`),
+        7: recordFamiliarIOTM($familiar`Melodramedary`),
+        8: recordItemIOTM($item`packaged SpinMaster™ lathe`, $item`SpinMaster™ lathe`),
+        9: recordItemIOTM($item`Bagged Cargo Cultist Shorts`, $item`Cargo Cultist Shorts`),
+        10: recordSkillIOTM($skill`Comprehensive Cartography`, $item`Comprehensive Cartographic Compendium`),
+        11: recordItemIOTM($item`packaged knock-off retro superhero cape`, $item`unwrapped knock-off retro superhero cape`),
+        // While cute, the ghosts need weird casing because it's 3-in-1.
+        12: [$item`box o' ghosts`, {
+            storeItem: $item`box o' ghosts`,
+            have: _have($familiar`Ghost of Crimbo Commerce`) || _have($familiar`Ghost of Crimbo Carols`) || _have($familiar`Ghost of Crimbo Cheer`),
+            count: availableAmount($item`box o' ghosts`),
+        }],
+    },
+    2021: {
+        1: recordItemIOTM($item`packaged miniature crystal ball`, $item`miniature crystal ball`),
+        2: recordSkillIOTM($skill`Emotionally Chipped`, $item`emotion chip`),
+        3: recordItemIOTM($item`power seed`, $item`potted power plant`),
+        4: recordItemIOTM($item`packaged backup camera`, $item`backup camera`),
+        5: recordFamiliarIOTM($familiar`Shorter-Order Cook`),
+        6: recordItemIOTM($item`packaged familiar scrapbook`, $item`familiar scrapbook`),
+        7: recordVIPIOTM($item`clan underground fireworks shop`),
+        8: recordEudoraIOTM($item`Our Daily Candles™ order form`),
+        9: recordItemIOTM($item`packaged industrial fire extinguisher`, $item`industrial fire extinguisher`),
+        10: recordFamiliarIOTM($familiar`Vampire Vintner`),
+        11: recordItemIOTM($item`packaged Daylight Shavings Helmet`, $item`Daylight Shavings Helmet`),
+        12: recordWorkshedIOTM($item`packaged cold medicine cabinet`, $item`cold medicine cabinet`),
+    },
+    2022: {
+        1: recordPreferenceIOTM($item`undrilled cosmic bowling ball`,"hasCosmicBowlingBall"),
+        2: recordItemIOTM($item`combat lover's locket lockbox`, $item`combat lover's locket`),
+        3: recordFamiliarIOTM($familiar`Grey Goose`),
+        // I am not sure this will work for the umbrella honestly.
+        4: recordFoldableIOTM($item`undamaged Unbreakable Umbrella`),
```

Comment:
It's not foldable, it's just a single item stored as a single item id, so it's just an itemIoTM